pkg/virtctl/imageupload/imageupload.go

```diff
@@ -146,35 +121,35 @@ func init() {
-	cmd.Flags().StringVar(&size, "size", "", "The size of the DataVolume to create (ex. 10Gi, 500Mi).")
-	cmd.Flags().StringVar(&storageClass, "storage-class", "", "The storage class for the PVC.")
-	cmd.Flags().StringVar(&accessMode, "access-mode", "", "The access mode for the PVC.")
-	cmd.Flags().BoolVar(&blockVolume, "block-volume", false, "Create a PVC with VolumeMode=Block (default is the storageProfile default. for archive upload default is filesystem).")
-	cmd.Flags().StringVar(&volumeMode, "volume-mode", "", "Specify the VolumeMode (block/filesystem) used to create the PVC. Default is the storageProfile default. For archive upload default is filesystem.")
-	cmd.Flags().StringVar(&imagePath, "image-path", "", "Path to the local VM image.")
-	cmd.Flags().StringVar(&archivePath, "archive-path", "", "Path to the local archive.")
-	cmd.Flags().BoolVar(&noCreate, "no-create", false, "Don't attempt to create a new DataVolume/PVC.")
-	cmd.Flags().UintVar(&uploadPodWaitSecs, "wait-secs", 300, "Seconds to wait for upload pod to start.")
-	cmd.Flags().BoolVar(&forceBind, "force-bind", false, "Force bind the PVC, ignoring the WaitForFirstConsumer logic.")
-	cmd.Flags().BoolVar(&dataSource, "datasource", false, "Create a DataSource pointing to the created DataVolume/PVC.")
-	cmd.Flags().StringVar(&defaultInstancetype, "default-instancetype", "", "The default instance type to associate with the image.")
-	cmd.Flags().StringVar(&defaultInstancetypeKind, "default-instancetype-kind", "", "The default instance type kind to associate with the image.")
-	cmd.Flags().StringVar(&defaultPreference, "default-preference", "", "The default preference to associate with the image.")
-	cmd.Flags().StringVar(&defaultPreferenceKind, "default-preference-kind", "", "The default preference kind to associate with the image.")
+	cmd.Flags().BoolVar(&c.insecure, "insecure", false, "Allow insecure server connections when using HTTPS.")
+	cmd.Flags().StringVar(&c.uploadProxyURL, "uploadproxy-url", "", "The URL of the cdi-upload proxy service.")
+	cmd.Flags().StringVar(&c.name, "pvc-name", "", "The destination DataVolume/PVC name.")
+	cmd.Flags().StringVar(&c.pvcSize, "pvc-size", "", "The size of the PVC to create (ex. 10Gi, 500Mi).")
+	cmd.Flags().StringVar(&c.size, "size", "", "The size of the DataVolume to create (ex. 10Gi, 500Mi).")
+	cmd.Flags().StringVar(&c.storageClass, "storage-class", "", "The storage class for the PVC.")
+	cmd.Flags().StringVar(&c.accessMode, "access-mode", "", "The access mode for the PVC.")
+	cmd.Flags().BoolVar(&c.blockVolume, "block-volume", false, "Create a PVC with VolumeMode=Block (default is the storageProfile default. for archive upload default is filesystem).")
+	cmd.Flags().StringVar(&c.volumeMode, "volume-mode", "", "Specify the VolumeMode (block/filesystem) used to create the PVC. Default is the storageProfile default. For archive upload default is filesystem.")
```

Comment:
Aren't these 2 option redundant?

Reply:
If you don't want to change the command line, then please use a local variable and then simply set the `volumeMode` in the sturct. I think we can simplify and reduce the number of field defined there

Reply:
> Aren't these 2 option redundant?

How we can check it? When I inspect into `PersistentVolumeClaimSpec` I don't see `VolumeMode` as deprecated.

Reply:
> If you don't want to change the command line, then please use a local variable and then simply set the volumeMode in the sturct. I think we can simplify and reduce the number of field defined there

I see that both `--block-volume` and `--volume-mode` is used in the unit tests. I'd prefer not to remove or modify it in this PR, shall we do it in a new PR?

Reply:
Yes, sounds good